src/pages/NewChatPage.js

```diff
@@ -143,8 +140,15 @@ class NewChatPage extends Component {
     /**
      * Once all our options are selected this method will call the API and  create new chat between all selected users
      * and the currently logged in user
+     *
+     * @param {Object} option
      */
-    createGroup() {
+    openChatOrCreateGroup(option) {
```

Comment:
**suggestion:** Use two methods. We are giving this method an extra responsibility. Maybe convenient based on the caller, but harder to understand now.

Reply:
Cleaned up callbacks in this component